packit_service/worker/tasks.py

```diff
@@ -333,6 +342,52 @@ def run_retrigger_bodhi_update(
     return get_handlers_task_results(handler.run_job(), event)
 
 
+@celery_app.task(
+    bind=True,
+    name=TaskName.vm_image_build,
+    base=HandlerTaskWithRetry,
+    queue="long-running",
```

Comment:
since the repo doesn't need to be cloned (and shouldn't be), I think we can go for the short-running queue